app/models/job_invocation.rb

```diff
@@ -204,17 +204,13 @@ def output(host)
   def generate_description
     key_re = /%\{([^\}]+)\}/
     template_invocation = pattern_template_invocations.first
-    hash_base = Hash.new { |hash, key| hash[key] = "%{#{key}}" }
-    input_hash = template_invocation.input_values.reduce(hash_base) do |h, v|
+    input_hash = template_invocation.input_values.reduce({}) do |h, v|
       h.update(v.template_input.name => v.value)
     end
     input_hash.update(:job_category => job_category)
     input_hash.update(:template_name => template_invocation.template.name)
```

Comment:
I think this should fix the tests:
```suggestion
    input_hash.update("job_category" => job_category)
    input_hash.update("template_name" => template_invocation.template.name)
```